src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -283,120 +288,112 @@ private String getAsyncResultEndpoint(String jobId) {
         samService.listRoleBindings(id, getAuthenticatedInfo());
-    RoleBindingList responseList = new RoleBindingList();
+    ApiRoleBindingList responseList = new ApiRoleBindingList();
     for (bio.terra.workspace.service.iam.model.RoleBinding roleBinding : bindingList) {
       responseList.add(
-          new bio.terra.workspace.generated.model.RoleBinding()
-              .role(roleBinding.role().toApiModel())
-              .members(roleBinding.users()));
+          new ApiRoleBinding().role(roleBinding.role().toApiModel()).members(roleBinding.users()));
     }
     return new ResponseEntity<>(responseList, HttpStatus.OK);
   }
 
   @Override
-  public ResponseEntity<CreateCloudContextResult> createCloudContext(
-      UUID id, @Valid CreateCloudContextRequest body) {
-    ControllerValidationUtils.validateCloudPlatform(body.getCloudType());
+  public ResponseEntity<ApiCreateCloudContextResult> createCloudContext(
+      UUID id, @Valid ApiCreateCloudContextRequest body) {
+    ControllerValidationUtils.validateCloudPlatform(body.getCloudPlatform());
     AuthenticatedUserRequest userReq = getAuthenticatedInfo();
     String jobId = body.getJobControl().getId();
     String resultPath = getAsyncResultEndpoint(jobId);
 
     // For now, the cloud type is always GCP and that is guaranteed in the validate.
     workspaceService.createGcpCloudContext(id, jobId, resultPath, userReq);
-    CreateCloudContextResult response = fetchCreateCloudContextResult(jobId, userReq);
+    ApiCreateCloudContextResult response = fetchCreateCloudContextResult(jobId, userReq);
     return new ResponseEntity<>(
         response, HttpStatus.valueOf(response.getJobReport().getStatusCode()));
   }
 
   @Override
-  public ResponseEntity<CreateCloudContextResult> getCreateCloudContextResult(
+  public ResponseEntity<ApiCreateCloudContextResult> getCreateCloudContextResult(
       UUID id, String jobId) {
     AuthenticatedUserRequest userReq = getAuthenticatedInfo();
-    CreateCloudContextResult response = fetchCreateCloudContextResult(jobId, userReq);
+    ApiCreateCloudContextResult response = fetchCreateCloudContextResult(jobId, userReq);
     return new ResponseEntity<>(
         response, HttpStatus.valueOf(response.getJobReport().getStatusCode()));
   }
 
-  private CreateCloudContextResult fetchCreateCloudContextResult(
+  private ApiCreateCloudContextResult fetchCreateCloudContextResult(
       String jobId, AuthenticatedUserRequest userReq) {
     final AsyncJobResult<GcpCloudContext> jobResult =
         jobService.retrieveAsyncJobResult(jobId, GcpCloudContext.class, userReq);
 
-    final GcpContext gcpContext;
-    if (jobResult.getJobReport().getStatus().equals(StatusEnum.SUCCEEDED)) {
-      gcpContext = new GcpContext().projectId(jobResult.getResult().getGcpProjectId());
+    final ApiGcpContext gcpContext;
+    if (jobResult.getApiJobReport().getStatus().equals(StatusEnum.SUCCEEDED)) {
```

Comment:
I don't know if you wanted names inside functions to be changed or not.

Reply:
No, I tried not to do that, but it happened a time or three. Fixed it.